torch/_dynamo/variables/torch.py

```diff
@@ -597,68 +597,20 @@ def fn_with_prim_types(x):
             return UserFunctionVariable(
                 torch.nn.init._calculate_correct_fan, **options
             ).call_function(tx, args, {})
-        elif self.value == torch.utils._pytree.tree_flatten:
-            if len(args) != 1:
-                unimplemented("Unsupported flatten with len(args) != 1")
-
-            flattened, spec = torch.utils._pytree.tree_flatten(args[0])
-            return TupleVariable(
-                [ListVariable(flattened), ConstantVariable(spec)], **options
-            )
-        elif self.value == torch.utils._pytree.tree_unflatten:
-            if len(args) != 2:
-                unimplemented("Unsupported unflatten with len(args) != 2")
-
-            unflattened = torch.utils._pytree.tree_unflatten(args[0], args[1].value)
-
-            def _wrap_in_dynamo_variables(container):
-                if isinstance(container, VariableTracker):
-                    return container
-
-                if isinstance(container, list):
-                    return ListVariable(
-                        [_wrap_in_dynamo_variables(elem) for elem in container],
-                        **options,
-                    )
-
-                if isinstance(container, tuple):
-                    return TupleVariable(
-                        [_wrap_in_dynamo_variables(elem) for elem in container],
-                        **options,
-                    )
-
-                if isinstance(container, dict):
-                    return ConstDictVariable(
-                        {k: _wrap_in_dynamo_variables(v) for k, v in container.items()},
-                        type(container),
-                        **options,
-                    )
-
-            return _wrap_in_dynamo_variables(unflattened)
-
-        elif self.value == torch.fx._pytree.tree_flatten_spec:
-            if len(args) != 2:
-                unimplemented("Unsupported flatten_spec with len(args) != 2")
-
-            flattened, spec = torch.fx._pytree.tree_flatten_spec(args[0], args[1].value)
-            return TupleVariable(
-                [ListVariable(flattened), ConstantVariable(spec)], **options
-            )
-        elif self.value == torch.utils._pytree.tree_map_only:
-            if len(args) != 3:
-                unimplemented("Unsupported tree_map_only with len(args) != 3")
-
-            ty = args[0].value  # type
-            fn = args[1]  # map fn
-            tree = args[2]  # tree
-
-            def map_fn(v):
-                if ty == v.python_type():
-                    return fn.call_function(tx, [v], {})
-                else:
-                    return v
-
-            return torch.utils._pytree.tree_map(map_fn, tree)
+        elif (
+            hasattr(self.value, "__module__")
+            and self.value.__module__ == "torch.utils._pytree"
```

Comment:
It doesn't seem to me that you should need the case here. Since you just want to inline it, you want to arrange for these functions in pytree to have been wrapped as user function to begin with, and the call function would hit it automatically. This is a skipfiles thing. @yanboliang can give more advice.